docs/tutorial.rst

```diff
@@ -1,13 +1,23 @@
-Getting Started
-================
+QUARK: A Framework for Quantum Computing Application Benchmarking
+=================================================================
 
-Let's see how to run the framework. You can try out the framework without having a quantum computer at your disposal by
-relying on simulators.
+Quantum Computing Application Benchmark (QUARK) is a framework for orchestrating benchmarks of different industry applications on quantum computers.
+QUARK supports various applications, like the traveling salesperson problem (TSP), the maximum satisfiability (MaxSAT) problem, or the robot path optimization in the PVC sealing use case.
+It also features different solvers (e.g., simulated /quantum annealing and the quantum approximate optimization algorithm (QAOA)), quantum devices (e.g., IonQ and Rigetti), and simulators.
+It is designed to be easily extendable in all of its components: applications, mappings, solvers, devices, and any other custom modules.
+
+Paper
```

Comment:
This part is not necessary in the tutorial on readthedocs since there is a dedicated section for that on the [welcome page](https://github.com/QUARK-framework/QUARK/blob/dev/docs/index.rst).